metrics/ApplicationSecurity/NoKnownVulnerabilities/NoKnownVulnerabilites.yaml

```diff
@@ -0,0 +1,9 @@
+--- # Metadata
+- id: NoKnownVulnerabilities
+- description: This rule assesses whether a [Resource] has its [p1:vulnerability] property set correctly.
```

Comment:
Currently, only the Library Resource has the property vulnerabilities, right? Why not make it concrete?

Also, I suggest to phrase it a bit differently, e.g. _This rules assesses whether a [Library] has the expected value of [p1:vulnerability]_.

Reply:
The following 2 resources have the property _vulnerability_, which is why I did not specify it further:
- SecurityAdvisoryDocument
- Library